docs/examples/contrib/sqlalchemy/plugins/tutorial/full_app_with_init_plugin.py

```diff
@@ -1,17 +1,17 @@
 from collections.abc import AsyncGenerator
 from typing import Optional
 
+from advanced_alchemy.extensions.litestar.plugins import (
```

Comment:
```suggestion
from litestar.plugins.sqlalchemy import (
```

Any reason not to use these so it's under a single package namespace?